### DISCUSSION TITLE
Review bottleneck

### DISCUSSION THREAD

**Discussion by bhack** [keras-team/keras-cv#950] [General]
I start to have the impression that we are already approaching again to a review bottleneck as in many other community repositories.

I suggest that we invest a little bit more time on the governance strategy to engage the "external" community to the review and maintainership process:

https://github.com/pytorch/pytorch/pull/78850/files

**Comment by bhack** [comment]
IMHO the strategy is to find a solution to progressively improve these metrics:
https://chaoss.community/metric-inclusive-leadership/

**Comment by LukeWood** [comment]
Our primary goal with updating the RoadMap: https://github.com/keras-team/keras-cv/blob/master/.github/ROADMAP.md

and narrowing our focus to only welcome contributions guiding us towards these is to improve on this - and keep the repository in a better state.  Hopefully this helps, as I agree with you that at this moment review latencys and such are too high.

**Comment by bhack** [comment]
Yes but the wish of growing he contributors doesn't scale if you will not progressively increase the mentioned inclusion metrics.
The more one manages to nourish inclusiveness, the easier it will be to have [not exclusively occasional contributors](https://chaoss.community/metric-occasional-contributors/).

See the last two graphs at:
https://cauldron.io/project/5989?from_date=2021-10-20&to_date=2022-10-20&tab=community-reviews

And the metrics at:
https://cauldron.io/project/5989?from_date=2022-04-20&to_date=2022-10-20&tab=performance-reviews

So the narrowing of the roadmap IMHO is just one singe factor of the problem.
